redfish-core/lib/managers.hpp

```diff
@@ -2206,8 +2213,10 @@ inline void requestRoutesManager(App& app)
             if (oem)
             {
                 std::optional<nlohmann::json> openbmc;
+                std::optional<nlohmann::json> ibmOem;
                 if (!redfish::json_util::readJson(*oem, asyncResp->res,
-                                                  "OpenBmc", openbmc))
+                                                  "OpenBmc", openbmc, "IBM",
```

Comment:
It would be best if we had behind the flag too: 
I.e.  
 #ifdef BMCWEB_ENABLE_IBM_USB_CODE_UPDATE
 "IBM", ibmOem 
  #endif

Reply:
Done